backend/src/main/java/hal/JbertControlButtonName.java

```diff
@@ -0,0 +1,8 @@
+package hal;
+
+public class JbertControlButtonName {
```

Comment:
Shall the control and playlistselections button positions and gpio names be configurable in future?

Reply:
Not sure yet, might be interesting as soon as there is more than one hardware :)
I've changed the class to a enum anyway.